### DISCUSSION TITLE
Syntax error, unexpected T_FUNCTION in core.php

### DISCUSSION THREAD

**Discussion by rainkarnichi** [datenstrom/community#23] [Report a bug] [answered]
I don't know what happened.
Just get error:
`Parse error: syntax error, unexpected T_FUNCTION in /home/test000/public_html/system/core/core.php on line 703`

**Comment by markseuffert** [comment]
Can you tell me which PHP version you have?

**Comment by rainkarnichi** [comment]
oh. I have PHP version 5.2.17.
I did not check the PHP version before. 
yellowcms requires PHP version 5.3, right?

**Comment by markseuffert** [comment, marked as answer]
Yes, you're right. I am afraid there's not much I can do, Yellow requires PHP 5.3 or later.
Perhaps you can upgrade to a [newer PHP version](https://en.wikipedia.org/wiki/PHP#Release_history)? Thanks  for reporting the issue!
